pavelib/utils/test/suites/bokchoy_suite.py

```diff
@@ -55,9 +61,32 @@ def __enter__(self):
             test_utils.clean_test_files()
 
         msg = colorize('green', "Checking for mongo, memchache, and mysql...")
-        print(msg)
+        print msg
         bokchoy_utils.check_services()
 
+        if not self.testsonly:
+            self.prepare_bokchoy_run()
+
+        msg = colorize('green', "Confirming servers have started...")
```

Comment:
The msg would make more sense to me in both contexts if it instead said "Confirming servers are running..."